source/base/Profiler.ooc

```diff
@@ -51,8 +48,6 @@ Profiler: class {
 			}
 		this _timer free()
 		this _timer = null
```

Comment:
In that case, since it's just a one-line removal, and this is a cleanup PR, please delete it here.